docs/dev/install.rst

```diff
@@ -0,0 +1,135 @@
+Developer Setup
+===============
+The following describes how to set up an instance of the site on your
+computer for development.
+
+Prerequisites
+-------------
+This guide assumes you have already installed and set up the following:
+
+1. Git_
+2. `Python 3.5`_, `pip 8`_ or higher, and virtualenv_
+3. `Postgres 9.4`_
+4. ``openssl``
+
+These docs assume a Unix-like operating system, although the site should, in
+theory, run on Windows as well. All the example commands given below are
+intended to be run in a terminal.
+
+.. _Git: https://git-scm.com/
+.. _Python 2.7: https://www.python.org/
+.. _pip 8: https://pip.pypa.io/en/stable/
+.. _virtualenv: https://virtualenv.pypa.io/en/latest/
+.. _Postgres 9.4: http://www.postgresql.org/
+
+Installation
+------------
+1. Clone this repository or your fork_:
+
+   .. code-block:: bash
+
+      git clone --recursive https://github.com/mozilla/normandy.git
```

Comment:
We don't need `--recursive` (and we probably shouldn't have it, since it implies things that are wrong).